Source/Fuse.Nodes/Visual.Children.LinkedList.uno

```diff
@@ -0,0 +1,216 @@
+using Uno;
+using Uno.Collections;
+using Uno.Text;
+using Uno.UX;
+
+using Fuse.Internal;
+
+namespace Fuse
+{
+	public partial class Node
+	{
+		internal Node Children_next;
+		internal Node Children_previous;
+		internal int Children_parentID = -1;
+	}
+
+	public partial class Visual
+	{
+		static int Children_thisIDEnumerator;
+		readonly int Children_thisID = Children_thisIDEnumerator++;
+
+		void Children_MakeParent(Visual parent, Node child)
+		{
+			if (child.Children_parentID != -1) throw new Exception();
+			child.Children_parentID = parent.Children_thisID;
+		}
+
+		void Children_MakeOrphan(Node child)
+		{
+			child.Children_parentID = -1;
+		}
+
+		internal Node Children_first;
```

Comment:
As this is a mixin-like pattern, I use this naming convention to show that something fishy is going on. I'm thinking of this as a field of the `Children` mixin.

These symbols are not meant to be used outside of the file implementing the "mixin", I will refactor this to use proper properties instead.